webapp/util.go

```diff
@@ -12,59 +12,39 @@ import (
 	models "github.com/w3c/wptdashboard/shared"
 )
 
-var browsers map[string]models.Browser
-
-// No 'set' type in Go, so use map instead.
-var browserNames map[string]bool
-var browserNamesAlphabetical []string
-
-// GetBrowsers loads, parses and returns the set of names of browsers
-// which are to be included (flagged as initially_loaded in the JSON).
-func GetBrowsers() (map[string]models.Browser, error) {
-	if browsers != nil {
-		return browsers, nil
-	}
-	var bytes []byte
-	var err error
-	if bytes, err = ioutil.ReadFile("browsers.json"); err != nil {
-		return nil, err
-	}
-
-	if err = json.Unmarshal(bytes, &browsers); err != nil {
-		return nil, err
-	}
-	return browsers, nil
-}
+var browserNames, browserNamesAlphabetical = loadBrowsers()
```

Comment:
nit: Here or above `loadBrowsers()` implementation, call out that any error is considered fatal. @lukebjerring do you know what AppEngine will do if instances keep panicking on startup? I assume it gives up after a few retries and somehow flags the faulty deployed version, but I've never encountered this before.

Reply:
Locally, it seems to try a (large, but) finite number of times and then fall back to serving a copy of the failing stacktrace as a 500 for all requests. I think a unit test to ensure it parses is sufficient to catch this issue before prod..